bin/init.js

```diff
@@ -81,6 +98,81 @@ rl.on( 'close', () => {
 	process.exit( 0 );
 } );
 
+/**
+ * Run husky setup
+ *
+ * @return {void}
+ */
+const runHuskySetup = () => {
+	console.log( info.success( '\nInstalling husky...' ) );
+
+	// Check if .git file exists.
+	const gitDir = path.resolve( getRoot(), '.git' );
+	if ( ! fs.existsSync( gitDir ) ) {
+		console.log( info.warning( '\n.git directory does not exists.\nExiting without installing husky.' ) );
+		return;
+	}
+
+	// Run husky install command.
+	const huskyInstallCommand = `husky install`;
+	exec( huskyInstallCommand )
+		.then( ( result ) => {
+			console.log( info.success( `stdout: ${ result.stdout }` ) );
+			console.log( info.success( '\nHusky installed successfully!' ), '✨' );
+		} )
+		.catch( ( error ) => {
+			console.log( info.error( `error: ${ error.message }` ) );
+			if ( error.stderr ) {
+				console.log( info.error( `stderr: ${ error.stderr }` ) );
+			}
+		} );
+};
+
+/**
+ * Initialize Git.
+ *
+ * @return {Promise<void>}
+ */
+const initializeGit = async () => {
+	// Initialize git.
+	console.log( info.success( '\nInitializing git...' ) );
+
+	// Check if .git file exists.
+	const gitDir = path.resolve( getRoot(), '.git' );
+	if ( fs.existsSync( gitDir ) ) {
```

Comment:
Yes, So it will gracefully exit the process of initialization with the proper message. I am not understanding what is the concern here?

Reply:
@SH4LIN When initializing the theme for the first time, and theme is being cloned from GitHub in that case `.git` folder will always be there with the version control history of this repo and that's huge.

So if someone is setting up the for the first time && it already has a `.git` folder then it's not what they need. In this case, always delete the old `.git` folder and initialize a new one if the user has opted for it.

Also update the question for initializing the git as:
```
Would you like to initialize git (Note: It will delete any `.git` folder already in current directory)? (y/n)
```